rex/utilities/utilities.py

```diff
@@ -557,3 +558,27 @@ def get_lat_lon_cols(df):
         raise RuntimeError(msg)
 
     return lat_lon_cols
+
+
+def roll_timeseries(arr, timezones):
+    """
+    Roll array with unique shifts for each column
+    This converts timeseries to local time
+
+    Parameters
+    ----------
+    arr : ndarray
+        Input timeseries array of form (time, sites)
+    timezones : ndarray | list
+        Vector of timezone shifts from UTC to local time
+
+    Returns
+    -------
+    local_arr : ndarray
+        Array shifted to local time
+    """
+    local_arr = np.zeros(arr.shape, dtype=arr.dtype)
+    for i, s in enumerate(timezones):
+        local_arr[:, i] = np.roll(arr[:, i], int(s))
```

Comment:
Lets be aware that this will only work if the array is hourly data. Will not work for 5min WTK or NSRDB.